lib/AST/ASTContext.cpp

```diff
@@ -2692,6 +2692,12 @@ bool ASTContext::canImportModuleImpl(ImportPath::Module ModuleName,
     if (version.empty())
       return true;
 
+    // A versioned canImport, upon encountering an unversioned module
+    // for the queried identifier, evaluates to `true`.
```

Comment:
Should we only check one version based on `underlyingVersion`?

For example, an swift versioned canImport check should still succeed if there is a clang module overlay for the module which has underlying version?

Reply:
Good catch, thank you. Fixed.